src/optimise1.jl

```diff
@@ -0,0 +1,248 @@
+using IntervalRootFinding, IntervalArithmetic, StaticArrays, ForwardDiff, BenchmarkTools, Compat, IntervalOptimisation, DataStructures, IntervalConstraintProgramming
+
+import Base.isless
+
+struct IntervalMinima{T<:Real}
+    intval::Interval{T}
+    global_minimum::T
+end
+
+function isless(a::IntervalMinima{T}, b::IntervalMinima{T}) where {T<:Real}
+    return isless(a.global_minimum, b.global_minimum)
+end
+
+function minimise1d(f::Function, x::Interval{T}; reltol=1e-3, abstol=1e-3) where {T<:Real}
```

Comment:
Is this necessary any more?

Reply:
I think it's necessary for the derivative stuff, since that's where the difference lies. Should I combine them using runtime branching?